File: Backjoon2609/Backjoon2609.java
package Backjoon2609;

public class Backjoon2609{
    public static int gcd(int num1, int num2){
        if(num2 == 0) return num1;
        else return gcd(num2, num1 % num2);
    }
    public static void main(String args[]){
        int A = 24;
        int B = 18;
        int gcd = gcd(A, B);
        System.out.println(gcd);
        System.out.print(A * B / gcd);
    }
}

File: Backjoon2609/Backjoon2609.py
def gcd(num1, num2):
    if(num2 == 0):
        return num1
    else: 
        return gcd(num2, num1 % num2)


A = 24
B = 18
gcd = gcd(A, B)
print(gcd)
print(format(A * B / gcd, ".0f"))